docker/service/src/main/java/uk/nhs/adaptors/scr/models/gpsummarymodels/AllConditions.java

```diff
@@ -0,0 +1,12 @@
+package uk.nhs.adaptors.scr.models.gpsummarymodels;
+
+import java.util.List;
+
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class AllConditions {
```

Comment:
Why create a class just to hold a list?

Reply:
For Mustache template to only populate fields when this list is populated, if there is a better way to populate this let me know